packages/website/playwright.config.ts

```diff
@@ -5,7 +5,7 @@ const config: PlaywrightTestConfig = {
   forbidOnly: !!process.env.CI,
   fullyParallel: true,
   reporter: 'html',
-  retries: 0,
+  retries: 1,
```

Comment:
-1 from me on retries. In theory it's nice how it overrides small bits of flakiness, but in practice I think it allows flaky tests to stay undetected for longer. You end up with a lot of small levels of flake. I'd rather stamp out flake as soon as it's introduced.

Reply:
https://playwright.dev/docs/test-retries#retries